src/commands/edge/drivers/package.ts

```diff
@@ -118,7 +118,9 @@ $ smartthings edge:drivers:package -u driver.zip`]
 			processConfigFile(projectDirectory, zip)
 
 			processFingerprintsFile(projectDirectory, zip)
-			const testFileMatchers = buildTestFileMatchers(this.profileConfig.edgeDriverTestDirs)
+			// TODO: do more type checking here
+			const edgeDriverTestDirs = this.profile.edgeDriverTestDirs as string | string[]
```

Comment:
I'm going to make a `stringAarrayConfigValue` method in `SmartThingsCommand` in the next pull request for this.